### DISCUSSION TITLE
Unable to spin up pod using argocd rollout

### DISCUSSION THREAD

**Discussion by dhandma** [argoproj/argo-rollouts#951] [Q&A]
Hi All,
I have argocd v1.7 installed and on same cluster, I have installed the argocd-rollout latest binaries.

`

> 

kubectl-argo-rollouts: v0.10.1+840d463.dirty
  BuildDate: 2020-12-05T00:41:53Z
  GitCommit: 840d463b87684727302d71de7c464c2c099e924e
  GitTreeState: dirty
  GoVersion: go1.13.1
  Compiler: gc
  Platform: linux/amd64
`

Is there any compatibility issue between these two? I am not sure why I am not able to see the pods ? 
Also, when I am describing the rollout, I can see all fields are reflecting blank.
`

> 

[user1@server1 argocd-rollouts]$ kubectl describe rollout rollouts-demo
Name:         rollouts-demo
Namespace:    default
Labels:       <none>
Annotations:  <none>
API Version:  argoproj.io/v1alpha1
Kind:         Rollout
Metadata:
  Creation Timestamp:  2021-01-14T06:34:04Z
  Generation:          1
  Managed Fields:
    API Version:  argoproj.io/v1alpha1
    Fields Type:  FieldsV1
    fieldsV1:
      f:metadata:
        f:annotations:
          .:
          f:kubectl.kubernetes.io/last-applied-configuration:
      f:spec:
        .:
        f:replicas:
        f:revisionHistoryLimit:
        f:selector:
          .:
          f:matchLabels:
            .:
            f:app:
        f:strategy:
          .:
          f:canary:
            .:
            f:steps:
        f:template:
          .:
          f:metadata:
            .:
            f:labels:
              .:
              f:app:
          f:spec:
            .:
            f:containers:
    Manager:         kubectl-client-side-apply
    Operation:       Update
    Time:            2021-01-14T06:34:04Z
  Resource Version:  26859091
  Self Link:         /apis/argoproj.io/v1alpha1/namespaces/default/rollouts/rollouts-demo
  UID:               d7daeb29-a166-4764-8db6-84958d57e478
Spec:
  Replicas:                5
  Revision History Limit:  2
  Selector:
    Match Labels:
      App:  rollouts-demo
  Strategy:
    Canary:
      Steps:
        Set Weight:  20
        Pause:
        Set Weight:  40
        Pause:
          Duration:  10
        Set Weight:  60
        Pause:
          Duration:  10
        Set Weight:  80
        Pause:
          Duration:  10
  Template:
    Metadata:
      Labels:
        App:  rollouts-demo
    Spec:
      Containers:
        Image:  argoproj/rollouts-demo:blue
        Name:   rollouts-demo
        Ports:
          Container Port:  8080
          Name:            http
          Protocol:        TCP
        Resources:
          Requests:
            Cpu:     5m
            Memory:  32Mi
Events:              <none>
`

It is continuously showing me the progressing status. 
`
Name:            rollouts-demo
Namespace:       default
Status:          ◌ Progressing
Message:         more replicas need to be updated
Strategy:        Canary
  Step:
  SetWeight:     20
  ActualWeight:  0
Replicas:
  Desired:       5
  Current:       0
  Updated:       0
  Ready:         0
  Available:     0

NAME             KIND     STATUS         AGE  INFO
⟳ rollouts-demo  Rollout  ◌ Progressing  36s
`

Also, there is no update for get command also. 
`

> 

[user1@server1 argocd-rollouts]$ kubectl get rollout
NAME            DESIRED   CURRENT   UP-TO-DATE   AVAILABLE
rollouts-demo   5
`

Any help will be appreciated.

**Comment by Edblakejitsuin** [comment]
also seeing the same behaviour when trying out the rollouts-demo, stuck in progressing with no pods

**Comment by shrey-183** [comment]
I am seeing the same behavior too, any updates regarding this issue?

**Comment by jessesuen** [comment]
I think all of these exhibit the behavior of the controller not being installed in the cluster. Please note that if you choose a different namespace to install Rollouts in (from the install instructions), then you need to update the ClusterRoleBinding with the different namespace.
